Libraries/LibWeb/CSS/Length.cpp

```diff
@@ -137,6 +139,73 @@ CSSPixels Length::viewport_relative_length_to_px(CSSPixelRect const& viewport_re
     }
 }
 
+// https://drafts.csswg.org/css-conditional-5/#container-lengths
+CSSPixels Length::container_relative_length_to_px(Layout::Node const* layout_node, CSSPixelRect const& viewport_rect) const
+{
+    // FIXME: Percentages aren't resolved correctly in here yet because that would require us to reach
+    //        into the layout state to determine the container's parent's with or height.
+    //        There will probably be a better way to do this once these can get absolutized up-front.
```

Comment:
Nit: Can you make this FIXME a bit clearer about what needs to be done? I think you mean we need to read the query container's layout node's actual dimensions instead of the computed_values() ones?